doc/nrf/index.rst

```diff
@@ -3,11 +3,12 @@
 Welcome to the |NCS|!
 #####################
 
-The |NCS| contains libraries and application examples for Nordic Semiconductor devices. It uses the |link_zephyr|_ as real-time operating system.
+The |NCS| is where you begin building low power cellular IoT applications with LTE-M and NB-IoT on the nRF9160.
+It offers a range of application samples and reference implementations, as well as a full suite of device drivers for the nRF9160 SiP.
```

Comment:
isn't it literally one driver at this point - link control?

Reply:
This text is from Joel - I think it's okay since it's for the future anyway.